bridge/endpointActions/BarbellActions.cpp

```diff
@@ -170,8 +185,35 @@ httpd::Response configureBarbell(ActionContext* context,
             utils::format("Failed to configure barbell transport %s - %s", conferenceId.c_str(), barbellId.c_str()));
     }
 
-    mixer->addBarbellToEngine(barbellId);
+    std::vector<BarbellStreamGroupDescription> videoDescriptions;
+    auto& videoDescription = barbellDescription._video.get();
+    for (size_t i = 0; i < videoDescription._ssrcGroups.size(); i += 2)
```

Comment:
The fact that we're assuming correct interleaving of SSRCs is bothering me, it does not immediately screams "not robust" but it is close to. Why not to use dedicated arrays (e.g.: mediaSsrc, feedbackSsrc) and getters like you did for BarbellStreamGroupDescription?

Reply:
Totally agree. It hurts to leave it like this, but I created a ticket to refactor this specifically. It would create a lot of changes in EndpointDescripton etc to facilitate code reuse.